test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1721,6 +1721,8 @@ def tearDown(self):
 
         for mom in self.moms.values():
             mom.cleanup_files()
+            for u in PBS_USERS:
+                mom.cleanup_homedir(u.name)
```

Comment:
Should this be done in tearDown() or tearDownClass()?  Also, you str() the user in cleanup_homedir(), so you can pass 'u'

Reply:
I want this to be done in tearDown() to cleanup files after each test. Most performance tests submit more than thousands of jobs in each test.